app/src/main/java/org/oppia/app/testing/DragDropTestActivity.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.app.testing
+
+import android.os.Bundle
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.recyclerview.OnItemDragListener
+import javax.inject.Inject
+
+class DragDropTestActivity: InjectableAppCompatActivity(), OnItemDragListener {
```

Comment:
Add a brief KDoc here, as well please. In general, all public classes, constants, methods, and members should have KDocs unless they are truly trivial (generally only setters & getters fall into this category).

Reply:
done